src/commands/pull.js

```diff
@@ -14,12 +14,31 @@ var Config = require('../config');
 var Template = require('../template');
 
 module.exports = function (id) {
+
     var gitBranchPromise = Q.nfcall(exec, 'git rev-parse --abbrev-ref HEAD').catch(function (error) {
         console.log('This is not a git repo or there was an error getting the branch name', error);
     });
 
-    Q.all(gitBranchPromise)
+    var gitIssuesPromise = Q.nfcall(exec, 'hub issue').catch(function (error) {
+        console.log('This is not a git repo or there was an error getting the issues', error);
+    });
+    Q.all([gitBranchPromise, gitIssuesPromise])
         .then(function (results) {
+            var issuesAndPullRequests = _.filter(_.map(results[1][0].split('\n').sort(), function(i) {
+                return i.trim().replace(/(\d+)]([^\(]+)/, '#$1 [$2]');
```

Comment:
would be nice to have a comment with example of the issue as it comes back from hub explaining the regex